### ISSUE TITLE
allow filtering based on username

### ISSUE THREAD

**Issue by Stantheman** [Stantheman/fakehubstats#1]
Need to be smarter about grabbing commit history from git so I don't end up with points for commits I didn't make

**Comment by mdcollins05** [comment]
This would actually allow me to use this tool. I may take a try at this, although I've done very little Perl. :-/

**Comment by Stantheman** [comment]
@mdcollins05 Yeah, this feature is definitely big for any project that isn't solo. This issue really stands for more configuration of what gets tuned in this line:

https://github.com/Stantheman/fakehubstats/blob/master/fakehubstats.pl#L19

So it *should* be a git flag problem, and not a git problem. If you want to give it a shot, grab the tool, and change that line to add "--author=<whatever>" and see if that does it.
